### ISSUE TITLE
invio file al docente e invio file all'alunno

### ISSUE THREAD

**Issue by gmontalbano5ct** [scaforchio/LAMPSchool#73]
volevo chiedere se era possibile che l'alunno e il genitore potevano inviare dei file a tutti i docenti o ad un docente singolarmente. oppure il docente che invia un file a tutti gli alunni di una classe o ad un alunno singolarmente questa cosa si potrebbe fare?

**Comment by gmontalbano5ct** [comment]
Qualcuno mi può aiutare?

**Comment by gmontalbano5ct** [comment]
Qualcuno mi sa rispondere?

**Comment by valerio-bozzolan** [comment]
(Il modo per incoraggiare i volontari a dare una risposta più velocemente è di modificare il primo messaggio ulteriormente, ad esempio aggiungendo più dettagli. Se non è possibile, meglio attendere che ripetere la domanda più volte.)

La cosa è indubbiamente utile ma non inviterei a concentrare le energie per trasformare il registro elettronico in un sistema filesharing o chat. Oltretutto, alcune scuole sembra che non abbiano alcun tecnico di riferimento per gestire il registro, e caricare documenti sensibili nel registro potrebbe essere pericoloso per la singola scuola se non dovesse sapere come proteggere il suo sistema. 

Un modo veloce, semplice e sicuro sarebbe quello di semplicemente mostrare la mail del docente per permettere comunicazioni private e mandare file (link `mailto:`).

Ho bisogno di una mano per capire in quale situazione serva contattare _tutti_ i docenti. Forse ha più senso contattare il coordinatore di classe.

**Comment by gmontalbano5ct** [comment]
No io chiedevo di attivare la funzione fad cioè assegnare un compito anche senza scadenza e gli alunni possono inviare gli esercizi svolti

**Comment by scaforchio** [comment]
Direi che la funzinalità FAD andrebbe a snaturare la natura del registro. 
Per il FAD è molto meglio usare strumenti sviluppati per quello scopo tipo Moodle con il quale, tra l'altro, LAMPSchool si interfaccia per la gestione degli utenti e de corsi.

**Comment by vittodevit** [comment]
> Direi che la funzinalità FAD andrebbe a snaturare la natura del registro. Per il FAD è molto meglio usare strumenti sviluppati per quello scopo tipo Moodle con il quale, tra l'altro, LAMPSchool si interfaccia per la gestione degli utenti e de corsi.

Concordo, LAMPSchool e Moodle insieme sono quello che usiamo in produzione da noi, e ha sempre funzionato benissimo.
